frontend/src/components/ui/DrawflowWrapper.jsx

```diff
@@ -202,6 +202,8 @@ export default function DrawflowWrapper() {
   const dropPipeline = (pipelineData) => {
     const pipelineJson = JSON.parse(pipelineData);
     const { specs, path } = pipelineJson;
+    const newId = generateId(pipeline.name);
```

Comment:
Should use `pipeline.id` since name looks like `Canny Edge Demo` and spaces are not allowed in argo pipelines

Reply:
changed it:

`const newId = generateId(pipeline.id);`